### PR TITLE
Add neva chain

### PR DESCRIPTION
**Pull request by hyperchris123**

### CODE REVIEW

File: packages/config/src/projects/layer2s/neva.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { upcomingL2 } from './templates/upcoming'
+import { Layer2 } from './types'
+
+export const neva: Layer2 = upcomingL2({
+  id: 'neva',
+  display: {
+    name: 'Neva',
+    slug: 'neva',
+    description:
+      'Neva is an EVM-equivalent ZK-application network that provides scalability and security through Celestia DA and Polygon zkEVM, transforming GameFi and streamlining DeFi.',
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/neva.ts:10]:
```suggestion
      'Neva is an EVM-equivalent ZK-application network that provides scalability through Celestia DA and Polygon zkEVM, transforming GameFi and streamlining DeFi.',
```

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/projects/layer2s/neva.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export const neva: Layer2 = upcomingL2({
+  id: 'neva',
+  display: {
+    name: 'Neva',
+    slug: 'neva',
+    description:
+      'Neva is an EVM-equivalent ZK-application network that provides scalability through Celestia DA and Polygon zkEVM, transforming GameFi and streamlining DeFi.',
+    purposes: ['Universal', 'Gaming'],
+    category: 'ZK Rollup',
```
Comment (lucadonnoh) [inline packages/config/src/projects/layer2s/neva.ts:12]:
```suggestion
    category: 'Validium',
```
if it plans to use celestia

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

**Comment by hello-jdj** [comment]
Hi @lucadonnoh  have completed all the modifications. Thank you for your feedback!

**Review by lucadonnoh — APPROVED** [review]

**Comment by hyperchris123** [comment]
Hi @sekuba  I'm encountering a lint error, which seems similar to this one:
https://github.com/l2beat/l2beat/pull/5178#issuecomment-2363655110

Could you help me take a look?  When I run npm, I get this error, but the call is still being recorded in the Etherscan backend.

```
yarn check-verified-contracts 
yarn run v1.22.22
$ node -r esbuild-register scripts/checkVerifiedContracts
Check Verified Contracts.
=========================
ETHERSCAN_WORKERS=4 (can be changed via environment variable)
Checking on chain ethereum...
Loaded 1814 previously verified contracts.
21:00:32.062 INFO Processing 1819 addresses.
21:00:32.064 INFO Checking 0xF000A3B10e1920aDC6e7D829828e3357Fc5128A9...
Error: bad result from backend (method="getCode", params={"address":"0xF000A3B10e1920aDC6e7D829828e3357Fc5128A9","blockTag":"latest"}, result="Missing/Invalid API Key", error={"reason":"invalid hexlify value","code":"INVALID_ARGUMENT","argument":"value","value":"Missing/Invalid API Key"}, code=SERVER_ERROR, version=providers/5.7.2)
    at Logger.Logger.makeError (l2beat/node_modules/@ethersproject/logger/src.ts/index.ts:269:28)
    at Logger.Logger.throwError (l2beat/node_modules/@ethersproject/logger/src.ts/index.ts:281:20)
    at StaticJsonRpcProvider.<anonymous> (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1489:27)
    at step (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1:14)
    at Object.next (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1:14)
    at fulfilled (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'bad result from backend',
  code: 'SERVER_ERROR',
  method: 'getCode',
  params: {
    address: '0xF000A3B10e1920aDC6e7D829828e3357Fc5128A9',
    blockTag: 'latest'
  },
  result: 'Missing/Invalid API Key',
  error: Error: invalid hexlify value (argument="value", value="Missing/Invalid API Key", code=INVALID_ARGUMENT, version=bytes/5.7.0)
      at Logger.Logger.makeError (l2beat/node_modules/@ethersproject/logger/src.ts/index.ts:269:28)
      at Logger.Logger.throwError (l2beat/node_modules/@ethersproject/logger/src.ts/index.ts:281:20)
      at Logger.Logger.throwArgumentError (l2beat/node_modules/@ethersproject/logger/src.ts/index.ts:285:21)
      at hexlify (l2beat/node_modules/@ethersproject/bytes/src.ts/index.ts:252:19)
      at StaticJsonRpcProvider.<anonymous> (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1487:27)
      at step (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1:14)
      at Object.next (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1:14)
      at fulfilled (l2beat/node_modules/@ethersproject/providers/src.ts/base-provider.ts:1:14)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'invalid hexlify value',
    code: 'INVALID_ARGUMENT',
    argument: 'value',
    value: 'Missing/Invalid API Key'
  }
}
```

**Pull request merged**